lib/kubeclient.rb

```diff
@@ -43,11 +40,15 @@ def rest_client
           ssl_client_cert:  @ssl_options[:client_cert],
           ssl_client_key:   @ssl_options[:client_key]
         }
-
-        RestClient::Resource.new(@api_endpoint, options)
+        endpoint_with_version = URI.parse(@api_endpoint.to_s)
+        endpoint_with_version.path = endpoint_with_version.path << '/' \
+                                     << @api_version
```

Comment:
> This can be simplified (removing the URI parse part) with something like:
> 
> endpoint_with_version = @api_endpoint.merge(@api_endpoint.path + '/' + @api_version)
> 
> Same thing below.

sure